src/coffee/helpers/user-agent.coffee

```diff
@@ -0,0 +1,12 @@
+{ arch, platform } = require 'os'
+getPkg = require 'load-module-pkg'
+
+module.exports = (pkgName, pkgVersion) =>
+  throw new Error 'Package name and version is required' unless pkgName and pkgVersion
```

Comment:
Do we want to throw an error (that is fatal) if there is no version found? Maybe fall back to a user agent with a missing SDK version? Not sure...